### PR TITLE
hood: do not subtract-underflow on null base hash

### PR DESCRIPTION
**Pull request by pkova**
Ran into this one today.

### CODE REVIEW

File: pkg/base-dev/sur/hood.hoon [hunk 1]
```diff
@@ -140,7 +140,7 @@
   ?.  verb
     =/  base-hash  ?.(=(1 (lent meb)) <meb> <(head meb)>)
     :~  leaf/"/sys/kelvin:          {kul}"
-        leaf/"base hash ends in:     {(slag (sub (lent base-hash) base-hash)}"
+        leaf/"base hash ends in:     {(slag (sub (max 5 (lent base-hash)) base-hash)}"
```
Comment (MarcusMiguel) [inline pkg/base-dev/sur/hood.hoon:143]: Shouldn't this be:
```suggestion
        leaf/"base hash ends in:     {(slag (sub (max 5 (lent base-hash)) 5) base-hash)}"

```

Comment (MarcusMiguel) [inline pkg/base-dev/sur/hood.hoon:143]: Shouldn't this be:
```suggestion
        leaf/"base hash ends in:     {(slag (sub (max 5 (lent base-hash)) 5) base-hash)}"
```

**Comment by jfranklin9000** [comment]
There's another issue in `+report-vat` that I will PR for later today.  I will be using
```
::  +truncate-hash: get last 5 digits of hash and convert to tape
::
++  truncate-hash
  |=  hash=@uv
  ^-  tape
  (slag 2 <`@uv`(mod hash 0v1.00000)>)
```

**Comment by pkova** [comment]
Closing in favor of #6522

**Pull request closed without merging**